CONTRIBUTING.md

```diff
@@ -26,12 +26,36 @@ cp .env.dist .env
 docker-compose up
 ```
 
-Now you should be able to view the tool at [http://localhost:8042/](http://localhost:8042/)
+Now you should be able to view the tool at [http://localhost:8042/](http://localhost:8042/).
 If you need to change the port, edit `TOOLFORGE_DOCKER_PORT` in your `svgtranslate/.env` file.
 
-To run Composer commands on the server, connect to the container:
+The usual commands you'll need are as follows:
 
-    docker exec -it -u dev svgtranslate_web bash
+1. If you change anything in `assets/` you'll need to rebuild the asset files
+   (running `npm install` before the first time doing this, if you haven't already):
+
+       docker exec svgtranslate_assets npm install
+       docker exec svgtranslate_assets npm run assets
```

Comment:
You could have it do this in the when you do `docker-compose up` (i.e. add it to the command file). But that can make starting up slower, so pros and cons.

Reply:
Yes, I wondered about that. I think probably `npm install` at least should be there; have moved it.